samigo/samigo-app/src/webapp/js/authoring.js

```diff
@@ -860,3 +860,371 @@ function setAccessibilityAttributes() {
 		$(settingQuerryIdent + ", " + settingQuerryOption).first().attr("aria-describredby", helpBlockId);
 	}
 }
+class FormElement {
```

Comment:
Add an space here!

Reply:
Could this logic be extracted into its own file? Something like safe.exam.browser.js ?

Reply:
There is an `safeExamBrowser.js` for the delivery. we could have `settingsSafeExamBrowser.js` / `authoringSafeExamBrowser.js` and `deliverySafeExamBrowser.js`.
But I feel like other parts of `authoring.js` could use the FormElement class. There are a lot of parts in the settings where we have similar logic show/hide logic using jQuery.

Reply:
But they are not going to use them for now, I'd like to have it in a separate file for compatibility reasons. Is it possible?

Reply:
sure!